docs/troubleshooting/password-reset.rst

```diff
@@ -15,11 +15,11 @@ Procedure
 
    sudo systemctl stop scylla-server
 
-| 2. Remove your tables under ``/var/lib/scylla/data/system_auth_v2/``.
+| 2. Remove your tables under ``/var/lib/scylla/data/system/role*``.
```

Comment:
This is read by a human, they might not understand the `*` syntax.
Maybe something like: "Remove your tables that start with the prefix ``role`` under ``/var/lib/scylla/data/system/``, i.e. ``roles``, ``role_permissions`` and so on.

cc @annastuchlik 

same below

Reply:
below is bash example so it should work, and here i'll add explanation